pkg/lint/lintersdb/custom_linters.go

```diff
@@ -115,8 +116,11 @@ func (m *Manager) lookupAnalyzerPlugin(plug *plugin.Plugin) ([]*analysis.Analyze
 		return nil, err
 	}
 
-	m.log.Warnf("plugin: 'AnalyzerPlugin' plugins are deprecated, please use the new plugin signature: " +
-		"https://golangci-lint.run/contributing/new-linters/#create-a-plugin")
+	// TODO(ldez): remove this env var in the next minor version (v1.55.0)
```

Comment:
You mean remove as in always show the message without support to surpress it or remove the whole log?

Reply:
I improved the comment